[DOCS-1407] Improve the accessibility of link text (#1438)
wandb/docs: content/en/launch/walkthrough.md

Based on your use case, you (or someone on your team) will configure the launch queue according to your chosen [compute resource target]({{< relref "./launch-terminology.md#target-resources" >}}) (for example Amazon SageMaker) and deploy a launch agent on your own infrastructure. 

> See the [Terms and concepts]({{< relref "./launch-terminology.md" >}}) page for launch jobs, queues, agents, and how W&B Launch works.

Author: ```suggestion
See the [Terms and concepts]({{< relref "./launch-terminology.md" >}}) page for more information about Launch.
```
